pkgs/by-name/li/libitl/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libitl";
+  version = "0.1.8";
+
+  nativeBuildInputs = [ cmake ];
+  outputs = [ "out" ];
+
+  src = fetchFromGitHub {
+    owner = "arabeyes-org";
+    repo = "ITL";
+    rev = "cf4a5555f492c6293efa1f8880d090c9f1769509";
+    hash = "sha256-Hp+NXibSlF9LMRRlHWISAiQSz/dbMNjjEfKCNFVmIxo=";
+  };
+
+  buildPhase = ''
+    cmake .
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/lib
+    mkdir -p $out/include/itl
+    cp libitl.so $out/lib/libitl.so.0
+    ln -s $out/lib/libitl.so.0 $out/lib/libitl.so
+    cp libitl.a $out/lib
+    cp $src/hijri/hijri.h $out/include/itl
+    cp $src/prayertime/prayer.h $out/include/itl
+  '';
```

Comment:
Please reuse upstream CMake [install instructions](https://github.com/arabeyes-org/ITL/blob/master/CMakeLists.txt#L45-L48). These can be patched with `substituteInPlace` if needed.

Reply:
Done.